.github/workflows/codeql-push.yml

```diff
@@ -6,7 +6,8 @@ permissions:
 on:
   push:
     paths:
-    - 'source/common/**'
+    - include/**
+    - source/common/**
```

Comment:
Could you either include the `'` delimiters like it used to, or also remove them from branches-ignore below, for consistency.

Reply:
ill remove them from below - that is more consistent with the rest of the code base